web/assets/js/widgets/core/snowplow-tracking.js

```diff
@@ -163,14 +163,13 @@
       entries.forEach((entry) => {
         if (!entry.isIntersecting) return;
 
-        const readMoreButtons = entry.target.getElementsByClassName(
-          "cnw_btn__card-readmore"
+        const cardTitle = entry.target.getElementsByClassName(
+          "cnw_card-title"
         );
 
-        if (!readMoreButtons[0]) return;
+        if (!cardTitle[0]) return;
 
-        const uri = readMoreButtons[0].href;
-        const url = new URL(uri);
+        const url = new URL(cardTitle[0].dataset.detailLink);
```

Comment:
Maybe add an intermediate variable for the link and add an early return if the link is not there?
You never know 🤷 

```suggestion
       const detailLink = cardTitle[0].dataset.detailLink
       if (!detailLink) return
        const url = new URL(detailLink);
```

Reply:
Solved with 58dab6f